src/cli.ts

```diff
@@ -1,51 +1,537 @@
+      group: 'URL Handling Options',
+      type: 'string',
+    })
+    // Auth Options
+    .option('basic-auth-password', {
+      description: 'basic http(s) auth password',
+      group: 'Auth Options',
+      type: 'string',
+    })
+    .option('basic-auth-username', {
+      description: 'basic http(s) auth username',
+      group: 'Auth Options',
+      type: 'string',
+    })
+    // Graphics Options
+    .option('disable-gpu', {
+      default: false,
+      description: 'disable hardware acceleration',
+      group: 'Graphics Options',
+      type: 'boolean',
+    })
+    .option('enable-es3-apis', {
+      default: false,
+      description: 'force activation of WebGL 2.0',
+      group: 'Graphics Options',
+      type: 'boolean',
+    })
+    .option('ignore-gpu-blacklist', {
+      default: false,
+      description: 'force WebGL apps to work on unsupported GPUs',
+      group: 'Graphics Options',
+      type: 'boolean',
+    })
+    // Caching Options
+    .option('clear-cache', {
+      default: false,
+      description:
+        'prevent the application from preserving cache between launches',
+      group: 'Caching Options',
+      type: 'boolean',
+    })
+    .option('disk-cache-size', {
+      defaultDescription: 'chromium default',
+      description:
+        'forces the maximum disk space (in bytes) to be used by the disk cache',
+      group: 'Caching Options',
+      type: 'number',
+    })
+    // (In)Security Options
+    .option('disable-old-build-warning-yesiknowitisinsecure', {
+      default: false,
+      description:
+        'Disables warning when opening an app made with an old version of Nativefier. Nativefier uses the Chrome browser (through Electron), and it is dangerous to keep using an old version of it.)',
+      group: '(In)Security Options',
+      type: 'boolean',
+    })
+    .option('ignore-certificate', {
+      default: false,
+      description: 'ignore certificate-related errors',
+      group: '(In)Security Options',
+      type: 'boolean',
+    })
+    .option('insecure', {
+      default: false,
+      description: 'enable loading of insecure content',
+      group: '(In)Security Options',
+      type: 'boolean',
+    })
+    // Flash Options (DEPRECATED)
+    .option('flash', {
+      default: false,
+      deprecated: true,
+      description: 'enables Adobe Flash',
+      group: 'Flash Options (DEPRECATED)',
+      type: 'boolean',
+    })
+    .option('flash-path', {
+      deprecated: true,
+      description: 'path to Chrome flash plugin; find it in `chrome://plugins`',
+      group: 'Flash Options (DEPRECATED)',
+      normalize: true,
+      type: 'string',
+    })
+    // Platform Specific Options
+    .option('app-copyright', {
+      description:
+        '(macOS, windows only) a human-readable copyright line for the app. Maps to `LegalCopyright` metadata property on Windows, and `NSHumanReadableCopyright` on macOS',
+      group: 'Platform Specific Options',
```

Comment:
Here and everywhere (or only once if you agree moving this to a const):

```suggestion
      group: 'Platform-Specific Options',
```